homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_zone_control.py

```diff
@@ -32,9 +33,40 @@ class AtlanticPassAPCZoneControl(OverkizEntity, ClimateEntity):
     )
     _enable_turn_on_off_backwards_compatibility = False
 
+    def __init__(
+        self, device_url: str, coordinator: OverkizDataUpdateCoordinator
+    ) -> None:
+        """Init method."""
+        super().__init__(device_url, coordinator)
+
+        # Cooling is supported by a separate command
+        if self.is_auto_hvac_mode_available:
+            self._attr_hvac_modes.append(HVACMode.AUTO)
```

Comment:
`_attr_hvac_modes` is now a mutable field with this change. That means it should be set in the constructor. Otherwise this will cause a bug where someone has 1 without auto hvac and 1 with, and then they share(!) the same list, which is a bug which is a pita